edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/publisher/LogAnalyticsUpload.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Publisher
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.AzureLogAnalytics;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public sealed class LogAnalyticsUpload : IMetricsPublisher
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<LogAnalyticsUpload>();
+        readonly string workspaceId;
+        readonly string workspaceKey;
+        readonly string logType;
+        readonly Guid guid;
+
+        public LogAnalyticsUpload(string workspaceId, string workspaceKey, string logType, Guid guid)
```

Comment:
please give a more meaningful name for guid argument

Reply:
I need to remove this guid